core/build.gradle

```diff
@@ -1,15 +1,21 @@
 sourceCompatibility = 1.7
 [compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
 
-sourceSets.main.java.srcDirs = [ "src/" ]
-project(":core") {
-    dependencies {
-        compile "com.badlogicgames.gdx:gdx-ai:1.8.1"
-        //classpath "com.badlogicgames.gdx:gdx-ai:1.8.1"
-    }
-}
+sourceSets.main.java.srcDirs = [ "src/main/java/" ]
+sourceSets.test.java.srcDirs = [ "src/test/java/" ]
+//sourceSets.test.resources.srcDirs = [ "src/test/res/" ]
```

Comment:
Why is all of core commented out?

Reply:
because that was in the wrong place. There's another gradle file where it's supposed to be. I've moved these to there

Reply:
is that the main build.gradle?

Reply:
nope

Reply:
It's the core build.gradle, as opposed to the main one or the desktop one